src/Unity.Mathematics.Editor/PostNormalizedVectorDrawer.cs

```diff
@@ -166,6 +184,7 @@ public override void OnGUI(Rect position, SerializedProperty property, GUIConten
             if (string.IsNullOrEmpty(label.tooltip))
                 label.tooltip = Content.tooltip;
 
+            guiData.RebuildIfDirty();
```

Comment:
This method was added to solve the problem of serialized data getting updated from a different context, and then being overwritten by the pre-normalized data cached by the drawer